packages/nvidia_nat_config_optimizer/src/nat/plugins/config_optimizer/eval_runtime_loader.py

```
@@ -0,0 +1,28 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import lru_cache
+
+
+@lru_cache(maxsize=1)
+def load_evaluation_run() -> type:
+    """Lazily load eval runtime class required by `nat optimize`."""
+    try:
+        from nat.plugins.eval.runtime.evaluate import EvaluationRun
+        return EvaluationRun
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)